redis.go

```diff
@@ -55,6 +65,7 @@ func newRedisLink(cfg serverCfg) (ret redisLink, err error) {
 		WriteTimeout: time.Millisecond * time.Duration(cfg.Redis.WriteTimeout),
 		DialTimeout:  time.Millisecond * time.Duration(cfg.Redis.DialTimeout),
 	})
+	ret.master.WrapProcess(instrumentRedisCmd)
```

Comment:
Is the performance impact of wrapping every redis command with a statsd call fairly negligible?

Reply:
Yes